APE18.rst

```diff
@@ -0,0 +1,69 @@
+Adopt NEP 29 for CPython and Numpy Version Support
+--------------------------------------------------
+
+author: Stuart Mumford and Thomas Robitaille
+
+date-created: 2020 October 16
+
+date-last-revised: 2020 October 16
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+After the release of astropy 4.0 APE 10 no longer specifies the supported versions of CPython.
+Since APE 10 was written the wider scientific Python community have written and largely adopted `NEP 29 <https://numpy.org/neps/nep-0029-deprecation_policy.html>`__ as a policy for supported versions of both CPython and Numpy.
+This APE proposes that Astropy follow the projects who have already adopted this policy.
+
+
+Detailed description
+--------------------
+
+This APE fully adopts NEP 29 for both CPython and Numpy versions, it supersedes APE 10 wherever there is a conflict between that APE and this APE or NEP 29.
+
+While this APE will not replicate the whole of NEP 29, and NEP 29 should be considered the canonical source for the support policy, the core scheme to be adopted is as follows:
+
+astropy will support:
+
+* All minor versions of Python released 42 months prior to the project, and at minimum the two latest minor versions.
+* All minor versions of numpy released in the 24 months prior to the project, and at minimum the last three minor versions.
+
+
+This means that for the next few releases of astropy the following versions of CPython and Numpy will be supported:
+
++--------------------+--------------------+-----------------+--------------------+
+| Astropy Release    | Release Date       | CPython Version | Numpy Version      |
++--------------------+--------------------+-----------------+--------------------+
+| 4.2                | November 2020      | 3.7+            | 1.16+              |
++--------------------+--------------------+-----------------+--------------------+
+| 5.0 (LTS)          | April 2021         | 3.7+            | 1.17+              |
```

Comment:
That doesn't seem right, 5.0 must be the following release? (4.0 was released in winter)

Reply:
https://github.com/astropy/astropy/wiki/Release-Calendar doesn't say. You sure we won't have 4.3?